### PR TITLE
setting landing-page/login view and starting dashboard homepage view

### PR DESCRIPTION
**Pull request by lcaloguerea**
[[NP-3](https://pappcorn.atlassian.net/secure/RapidBoard.jspa?rapidView=112&projectKey=NP&modal=detail&selectedIssue=NP-3)]

# Change description
- Created pages structure
- Created pages/login module
- Added UI elements and set routing to dashboard
- Created Dashboard module
- Added sidebar menu (In progress)

# Evidence
Landing Page / Login using minimum resolution established by design
imagen

Dashboard after login in progress
imagen

New folder structure for components
imagen

### CODE REVIEW

File: apps/dashboard/src/styles.scss [hunk 1]
```diff
@@ -1 +1,46 @@
...
+
 /* You can add global styles to this file, and also import other style files */
+
+html,
+body {
+  height: 100%;
+}
+body {
+  margin: 0;
+  font-family: Roboto, 'Helvetica Neue', sans-serif;
```
Comment (ruedagato) [inline apps/dashboard/src/styles.scss:45]: this style is incorrect, the real font was already imported

File: libs/styles/global.scss [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
 @import './colors';
 
-html, body {
+html,
+body {
   margin: 0;
   font-family: 'Inter', sans-serif;
 }
+
+.mat-form-field-appearance-fill .mat-form-field-flex {
```
Comment (ruedagato) [inline libs/styles/global.scss:10]: move this to a _material.scss file and import on this

**Review by ruedagato — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/app/app.component.html [hunk 3]
```diff
@@ -1,2 +1,3 @@
-<h1 class="text">Text primary 4</h1>
-<button pangButton>Test button</button>
+<!-- <h1 class="text">Text primary 4</h1>
```
[the remark this answers is not in the document]
Comment (lcaloguerea) [reply apps/dashboard/src/app/app.component.html:1]: Done

File: apps/dashboard/src/app/common-components/sidebar-menu/sidebar-menu.component.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
```
[the remark this answers is not in the document]
Comment (lcaloguerea) [reply apps/dashboard/src/app/common-components/sidebar-menu/sidebar-menu.component.ts:1]: Rule has been activated

File: apps/dashboard/src/app/common-components/sidebar-menu/sidebar-menu.component.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'new-pangea-sidebar-menu',
+  templateUrl: './sidebar-menu.component.html',
+  styleUrls: ['./sidebar-menu.component.scss']
+})
+export class SidebarMenuComponent implements OnInit {
```
[the remark this answers is not in the document]
Comment (lcaloguerea) [reply apps/dashboard/src/app/common-components/sidebar-menu/sidebar-menu.component.ts:9]: Move and build new folder structure 

- shared-components
       - components
             - sidebar-menu

File: apps/dashboard/src/app/pages/login/login.component.scss [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+      border-radius: 22px;
+      width: 400px;
+      height: 622px;
+      background-color: $buttonText;
+      box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.14);
+      padding: 0px 40px 0px 40px;
+      .forgot_password_container {
+        display: flex;
+        font-family: Inter;
+        font-size: 14px;
```
[the remark this answers is not in the document]
Comment (lcaloguerea) [reply apps/dashboard/src/app/pages/login/login.component.scss:43]: Added mixin structure

File: apps/dashboard/src/index.html [hunk 7]
```diff
@@ -6,8 +6,11 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
-  </head>
-  <body>
+      <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (lcaloguerea) [reply apps/dashboard/src/index.html:10]: Both fonts removed

File: libs/styles/global.scss (hunk 2 above)
Comment (lcaloguerea) [reply libs/styles/global.scss:10]: Moved

File: apps/dashboard/src/app/pages/login/login.component.html [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+            </mat-form-field>
+            <div class="forgot_password_container">
+                <p
+                  class="forgot_password_container__label"
+                  (click)="forgotPassword()"
+                >
+                  Forgot password
+                </p>
+              </div>
+              <button
```
Comment (ruedagato) [inline apps/dashboard/src/app/pages/login/login.component.html:48]: use pangButton like this example https://github.com/newPangea/mono-repo/blob/dev/libs/ui/src/lib/components/button/button.component.stories.ts#L8

**Review by ruedagato — CHANGES_REQUESTED** [review]

**Pull request merged**